docs/docs/framework-hooks/react.md

```diff
@@ -55,9 +53,29 @@ const MyComponent = () => {
 }
 ```
 
+### makePGliteProvider
+
+The `makePGliteProvider` function returns a `PGliteProvider` component and a `usePGlite` hook with the specified type, which enables you to provide a PGlite instance with all added extensions and retain then namespaces and types added to it.
+
+```ts
+import { PGlite } from '@electric-sql/pglite'
+import { live } from '@electric-sql/pglite/live'
+import { vector } from '@electric-sql/pglite/vector'
```

Comment:
We probably need the `LiveNamespace` and `VectorNamespace`, which may not exist.

alternative is :
```
const { PGliteProvider, usePGlite } = makePGliteProvider<
  PGlite & {
    live: typeof live
    vector: typeof vector
  }
>()
```

Reply:
Might be worth exporting the namespaces